test/unit.js

```diff
@@ -0,0 +1,273 @@
+import camelCase from 'camel-case';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactTestUtils from 'react-addons-test-utils';
+import reactify from '../src/index';
+import skate from 'skatejs';
+import helperElement from 'skatejs/test/lib/element';
+
+describe('react-integration;', function () {
+    var fixture;
```

Comment:
Also, instead of introducing side-effects through 'beforeEach` and `afterEach`, I've been trying to use functions that return what you need. That way you don't have to clean up, no side-effects and can use `const`.